### ISSUE TITLE
Update mnist tutorial

### ISSUE THREAD

**Issue by tomomano** [tomomano/learn-aws-by-coding#6]
- 以下の間違いを修正
```
testloader = torch.utils.data.DataLoader(trainset, batch_size=1000, shuffle=True)
```
- `test()` 関数を修正 => テストの loss, accuracy を return するように変更
- train data を train, val に分割する
